README.md

```diff
@@ -33,30 +33,44 @@ Prerequisites
     $ cd libtock-c
     ```
 
-1. The main requirement to build the C applications in this repository
-   is a cross compiler for embedded targets. Currently, the only
-   supportetd target is ARM Cortex M, so you will need an
-   `arm-none-eabi` toolchain.
+1. The main requirement to build the C applications in this repository is having
+   cross compilers for embedded targets. You will need an `arm-none-eabi`
+   toolchain for Cortex-M targets, and `riscv64-unknown-elf` toolchain for
+   RISC-V targets.
 
-   MacOS:
+   **MacOS**:
    ```
    $ brew tap ARMmbed/homebrew-formulae && brew update && brew install arm-none-eabi-gcc
+   $ brew tap riscv/riscv && brew update && brew install riscv-gnu-toolchain --with-multilib
    ```
 
-   Ubuntu (18.04LTS or later):
+   You can also download a pre-built RISC-V toolchain
+   [here](http://cs.virginia.edu/~bjc8c/archive/gcc-riscv64-unknown-elf-8.3.0-mac.zip).
+
+   **Ubuntu (18.04LTS or later)**:
    ```
    $ sudo apt install gcc-arm-none-eabi
    ```
 
-   Ubuntu (before 18.04):
+   You will also need a `riscv64` toolchain. You can download a pre-built
+   toolchain
+   [here](http://cs.virginia.edu/~bjc8c/archive/gcc-riscv64-unknown-elf-8.3.0-linux.zip),
```

Comment:
Can we also document the install guide for people who don't want to download this?

I'm unable to download it as well